README.md

```diff
@@ -1,23 +1,36 @@
-This base development image is used in conjunction with the [docker-cookiecutter-datascience](https://github.com/manifoldai/docker-cookiecutter-data-science)
-as part of the Orbyter toolset to help Data Science teams easily move to a container-first workflow from local development to serving in production settings. The goal of these
+# manifold/orbyter-docker 
+
+Manifold's Orbyter docker image toolset is to help Data Science teams easily move to a container-first workflow from local development to serving in production settings. The goal of these
 tools is to bring DevOps best practices to the Data Science community to increase productivity and quality of delivered worked to
 customers (internal and external).
 
-System/package info:
-
-- Ubuntu 16.04 (base image ubuntu:xenial-2018228)
-- Python 3.6.3
-- boto3 1.7.2
-- dask 0.17.2
-- distributed 1.21.6
-- jupyter 1.0.0
-- jupyter-contrib-nbextensions 0.5.0
-- keras 2.1.5
-- matplotlib 2.2.2
-- numpy 1.14.2
-- pandas 0.22.0
-- scikit-learn 0.19.1
-- scipy 1.0.1
-- tensorflow 1.6.0
-
-Additional Dockerfiles for different use cases will be added to this repository as work continues on the Orbyter project.
+## Docker images
+
+The various docker images are tagged by their repository name under their parent directory. E.g., The Dockerfile for docker-ml-dev:1.0 corresponds to docker-ml-dev/1.0/Dockerfile
+
+### docker-ml-dev
+
+This is currently the base development image used in conjunction with [docker-cookiecutter-datascience](https://github.com/manifoldai/docker-cookiecutter-data-science).
+
+### orbyter-base-sys
+
+Base docker image for ML development in python. ML images are build on top of this, and this should not be used directly for development.
```

Comment:
Ok this is clearer than the other PR. I would still want one more sentence to explain the design decision to factor the images into these two pieces.

Reply:
Updated in new push